### ISSUE TITLE
Unable to load library 'PCSC' on macOS Big Sur

### ISSUE THREAD

**Issue by Baco9** [intarsys/smartcard-io#9]
In macOS Big Sur, the system-provided libraries are not longer available on the file system.  When attempting to dlopen() a library path, it will look for the corresponding library in a cache instead of the file system.

The version of JNA that is currently used, does not seem to work correctly with this new behaviour. com.sun.jna.NativeLibrary.loadLibrary() now throws a java.lang.UnsatisfiedLinkError (from the _PCSC  constructor).

I fixed it by adding this snippet to my code:
`	NativePCSCLib pcscLib = new NativePCSCLib();
	pcscLib.setPath("/Library/Frameworks/PCSC.framework/PCSC");
	pcscLib.setUseExecutorThread(false);
	pcscLib.setUseBlockingGetStatusChange(false);
	pcscLib.initialize();
`
The better solution is probably to update the JNA dependency of the native-c project. I tested it with version 5.6.0, and there it works to simply load "PCSC", without a full path.

**Comment by eheck-is** [comment]
Yes, the latest JNA fixes it. Sorry, we didn't get around to updating the dependencies here yet.
